wallet/src/main/scala/org/bitcoins/wallet/api/WalletApi.scala

```diff
@@ -444,10 +446,27 @@ trait WalletApi extends WalletLogger {
       outPoints: Vector[TransactionOutPoint],
       address: BitcoinAddress,
       amount: CurrencyUnit,
-      feeRate: FeeUnit): Future[Transaction] = {
+      feeRateOpt: Option[FeeUnit],
+      fromAccount: AccountDb): Future[Transaction] = {
+    for {
+      feeRate <- feeRateOpt match {
```

Comment:
Let's refactor this `feeRateOpt match { ... } ` pattern into a helper method somewhere